builtin/providers/aws/resource_aws_elb.go

```diff
@@ -371,11 +386,47 @@ func resourceAwsElbUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange("idle_timeout") {
+		log.Println("[INFO] inside modify attributes")
+		attrs := elb.ModifyLoadBalancerAttributesInput{
+			LoadBalancerName: aws.String(d.Get("name").(string)),
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				ConnectionSettings: &elb.ConnectionSettings{
+					IdleTimeout: aws.Integer(d.Get("idle_timeout").(int)),
+				},
+			},
+		}
+		_, err := elbconn.ModifyLoadBalancerAttributes(&attrs)
+		if err != nil {
+			return fmt.Errorf("Failure configuring idle timeout: %s", err)
+		}
+		d.SetPartial("idle_timeout")
+	}
+
+	if d.HasChange("connection_draining") || d.HasChange("connection_draining_timeout") {
+		log.Println("[INFO] inside modify attributes")
+		attrs := elb.ModifyLoadBalancerAttributesInput{
+			LoadBalancerName: aws.String(d.Get("name").(string)),
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				ConnectionDraining: &elb.ConnectionDraining{
+					Enabled: aws.Boolean(d.Get("connection_draining").(bool)),
+					Timeout: aws.Integer(d.Get("connection_draining_timeout").(int)),
+				},
+			},
+		}
+		_, err := elbconn.ModifyLoadBalancerAttributes(&attrs)
+		if err != nil {
+			return fmt.Errorf("Failure configuring connection draining: %s", err)
+		}
+		d.SetPartial("connection_draining")
+		d.SetPartial("connection_draining_timeout")
+	}
+
```

Comment:
modifying both `idle_timeout` or `connection_draining` (and/or partner `connection_draining_timeout`) results in two `ModifyLoadBalancerAttributes` requests to AWS, but it looks like the API should support both of these in one request. Do you know if that's accurate? 

- http://docs.aws.amazon.com/ElasticLoadBalancing/latest/APIReference/API_ModifyLoadBalancerAttributes.html